cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -25,85 +22,139 @@ SRIOVDP_RESOURCE_NAME="sriov_net"
 VFS_DRIVER="vfio-pci"
 VFS_DRIVER_KMODULE="vfio_pci"
 
-function move_sriov_pfs_netns_to_node {
-  local -r node=$1
-  local -r pf_count_per_node=$2
-  local -r pid="$(docker inspect -f '{{.State.Pid}}' $node)"
-  local pf_array=()
+function prepare_node_netns() {
+  local -r node_name=$1
+  local -r node_pid=$(docker inspect -f '{{.State.Pid}}' "$node_name")
```

Comment:
Can we wrap the "docker" commands with functions like here `inspect_container $node_name` in case we move to podman or crioctl.

Reply:
If other container run-times has the same API we can simply create `CRI` variable that will be initialized from `kind/common.sh` or `provider.sh`.
IF not we probably need to create new script (that will be used as library) with all the warping functions, 
and source its on `kind/common.sh` or `provider.sh` script.

I think this change could affect other providers as well,
Would you mind if we do that on other PR?

Reply:
You are going to be safer with wrappers docker vs pod vs crioctl don't have full compatible interfaces, but you are right that this is not the place to address this.